plugins/modules/remote_management/stacki/stacki_host.py

```diff
@@ -258,8 +275,7 @@ def main():
             .format(module.params['name'])
     # Otherwise, state is present, but host doesn't exists, require more params to add host
     elif module.params['state'] == 'present' and not host_exists:
-        for param in ['appliance', 'prim_intf',
-                      'prim_intf_ip', 'network', 'prim_intf_mac']:
+        for param in ['appliance', 'rack', 'rank']:
```

Comment:
I would keep the two options in here, and treat them being ignored as a bug that should be fixed (not by this PR :) ).